documentation/modules/proc-configuring-kafka-connect-authentication-scram-sha-512.adoc

```diff
@@ -36,15 +44,21 @@ spec:
   # ...
   authentication:
     type: scram-sha-512
-    username: _MY-USERNAME_
+    username: _MY-CONNECT-USERNAME_
     passwordSecret:
-      secretName: _MY-SECRET_
-      password: _MY-PASSWORD.txt_
+      secretName: _MY-CONNECT-SECRET-NAME_
+      password: _PASSWORD-FIELD-NAME_
```

Comment:
Same as above. Adding a Secret YAML with the placeholder as the key might help.

Reply:
That might be confusing here because the user creates the Secret from a text file. Having example YAML for the Secret might suggest that they need to create a Secret from YAML.